src/main.rs

```diff
@@ -128,7 +128,7 @@ async fn main() {
                 Ok(Sk8brdMsgs::MsgPowerOn) => (),
                 Ok(Sk8brdMsgs::MsgPowerOff) => (),
                 Ok(Sk8brdMsgs::MsgFastbootPresent) => {
-                    if msg.len > 0 && buf[0] != 0 {
+                    if msg.len > 0 {
```

Comment:
NAK. On fastboot disconnect the server sends a packet with single '0' inside
```c
static void fastboot_disconnect(void *data)
{
        const uint8_t zero = 0;
    
        cdba_send_buf(MSG_FASTBOOT_PRESENT, 1, &zero);
} 
```

Reply:
OK my bad